File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    
    

    public UberVan(String license, Account driver, Map<String, 
    Map<String,Integer>> typeCarAccepted, 
    ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial; 
    }

    public UberVan(String license, Account driver){
    super(license, driver);
        
    }

    @Override
    public void setPassanger(Integer passenger) {
        if (passenger == 6 ){
            super.passenger = passenger;
            } else{
                System.out.println("Necesitas al menos 6 ocupantes");
            }
    }
}

File: PHP/uberVan.php
<?php
require_once('car.php');
class UberVan extends Car {
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial){
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatMaterial = $seatsMaterial; 
    } 
    public function setPassenger($passenger)
    {
        if($passenger == 6){
            $this->passenger = $passenger;
        } else{
            echo "<br> Este vehículo es para 6 personas<br>";
        }
    }
}
?>